packages/unchained-client/src/evm/arbitrum/parser/index.ts

```diff
@@ -10,7 +10,6 @@ import * as rfox from '../../parser/rfox'
 import * as zrx from '../../parser/zrx'
 
 export const ZRX_ARBITRUM_PROXY_CONTRACT = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
-export const RFOX_PROXY_CONTRACT_ADDRESS = '0xaC2a4fD70BCD8Bab0662960455c363735f0e2b56'
```

Comment:
Unused

Reply:
Feels like we could think about a contracts package as well that is a single source of truth for all things contracts. We have addresses spread all over the place, and also just noticing, we haven't update the parser with newest fox staking contracts and therefore aren't parsing new txs. Would help with scenarios such as that too.

Reply:
Absolutely agree about a contracts packages! We already have a `contracts` folder in web so should be an easy lift to just start by moving stuff out of it

Reply:
https://github.com/shapeshift/web/issues/7290